src/rest_server.cpp

```diff
@@ -151,32 +250,38 @@ void rest_server::doClassificationBatchPost(const Rest::Request &request,
   int count;
   float threshold;
   bool debugmode;
-  string language;
   string domain;
   
   try {
-    rest_server::fetchParamWithDefault(j, domain, language, count, threshold, debugmode);
+    rest_server::fetchParamWithDefault(j, domain, count, threshold, debugmode);
   } catch (std::runtime_error e) {
     response.headers().add<Http::Header::ContentType>(MIME(Application, Json));
     response.send(Http::Code::Bad_Request, e.what());
   }
   
-  tokenizer l_tok(language, true);
-  
   if (j.find("batch_data") != j.end()) {
     for (auto& it: j["batch_data"]){
       if (it.find("text") != it.end()) {
         string text = it["text"];
-        string tokenized = l_tok.tokenize_str(text);
-        it.push_back(nlohmann::json::object_t::value_type(
-            string("tokenized"), tokenized));
+        string tokenized;
+        it.push_back(nlohmann::json::object_t::value_type(string("tokenized"), tokenized));
```

Comment:
remove this line, tokenized hasn't been computed yet

Reply:
Done